tools/ete/ete_tree_generator.xml

```diff
@@ -0,0 +1,81 @@
+<tool id="species_tree_generator" name="species tree generator" version="0.1.1">
+    <description>Generates species tree from list of species using NCBI</description>
+     <requirements>
+        <requirement type="package">ete3</requirement>
+    </requirements>
+    <command>
+    <![CDATA[
+python $__tool_directory__/ete_tree_generator.py 
+-s $speciesFile
+#if $output_format.treebest == 'yes'
+    -f 8
+#else
+    -f ${output_format.format_selector}
+#end if
+-t $output_format.treebest
+> $outputFile
+    ]]>
+    </command>
+    <inputs>
+        <param name="speciesFile" type="data" format="text" label="Species file" help="Species file with list one in each line" />        
+        <conditional name="output_format">
+            <param name="treebest" type="select" label="Use in TreeBest" help="Select yes if specie tree to be used in TreeBest">
+                <option value="yes">Yes</option>
+                <option value="no" selected="true">No</option>
+            </param>
+            <when value="no">
+                <param name="format_selector" type="select" label="Output format">
+                    <option value="0" selected="true">0</option>
+                    <option value="1">1</option>
+                    <option value="2">2</option>
+                    <option value="3">3</option>
+                    <option value="4">4</option>
+                    <option value="5">5</option>
+                    <option value="6">6</option>
+                    <option value="7">7</option>
+                    <option value="8">8</option>
+                    <option value="9">9</option>
+                    <option value="100">100</option>
```

Comment:
I see now that you are providing the table in the help below, but I still think using `Description (number)` is nicer. Can you also add `help="See table below"` to `format_selector` ?